config/config.ts

```diff
@@ -0,0 +1,15 @@
+import * as fs from 'fs'
+
+let configFile
+let localConfig
+
+// If project is being run locally, the developer will have their API key in the json file
+try {
+  configFile = fs.readFileSync('./config/hollowbot.config.json', 'utf8')
+  localConfig = JSON.parse(configFile)
+} catch (error) {
+  // Project doesn't have a local config, we'll look for an environmental variable set via the Deployment medium
```

Comment:
The word "environment" in this context sounds more correct to me than "environmental" (I could be wrong). 

I don't think the word "Deployment" should be capitalized.